src/style.css

```diff
@@ -1,3 +1,107 @@
+html {
+  --header-height: 60px;
+  --active-link-color: rgba(0, 172, 220, 1);
+  --main-text-color: black;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  height: var(--header-height);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: var(--main-text-color);
+  text-transform: uppercase;
+  cursor: pointer;
+  display: flex;
+  margin-left: 20px;
+}
+
+.list__item {
+  line-height: var(--header-height);
+  position: relative;
+}
+
+.is-active {
+  display: flex;
+  flex-direction: column;
+  margin-left: 0;
+  color: var(--active-link-color);
+}
+
+.nav__link:hover {
+  color: var(--active-link-color);
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  background-color: var(--active-link-color);
+  border-radius: 8px;
```

Comment:
The border-radius should not be greater than half of the width and height.
In your case, it is effectively set to 2px by the browser.